src/ui/features/onboarding/Password/Password.tsx

```diff
@@ -70,10 +71,7 @@ function WeakPasswordWarning() {
           ref={focusNode}
           kind="primary"
           as={UnstyledLink}
-          onClick={(event) => {
-            event.preventDefault();
-            navigate(-1);
-          }}
+          onClick={goBack}
```

Comment:
Aren't we losing `preventDefault`?
Though I don't know what it was for, ideally we should comment

Reply:
I see there a `to="..."` prop lower down. `preventDefault` was obviously used to handle (prevent) that transaction, so it should be preserved

Reply:
ah, it has `as={UnstyledLink}`, I missed that somehow...